NEWS

```diff
@@ -237,7 +239,7 @@ What's new in psycopg 2.6.2
 - The `~psycopg2.extras.wait_select` callback allows interrupting a
   long-running query in an interactive shell using :kbd:`Ctrl-C`
   (:ticket:`#333`).
-- Fixed `!PersistentConnectionPool` on Python 3 (:ticket:`#348`).
+- Fixed ``PersistentConnectionPool`` on Python 3 (:ticket:`#348`).
```

Comment:
Why the format change? I prefer to use the

    `!Thing`

format for Python identifiers and

    ``thing``

for generic literals.